asset_dashboard/templates/asset_dashboard/partials/forms/add_edit_phase_form.html

```diff
@@ -48,47 +48,72 @@ <h3>{% if phase %}Update Phase{% else %}Create Phase{% endif %}</h3>
     <div class="d-flex justify-content-between m-3">
       <h3>Phase Funding</h3>
       {% if phase %}
-        <a href="{% url 'create-funding' pk=phase.id %}" class="text-info lead">Add funding ></a>
+        <a href='#/' class='btn btn-info' onclick='submitFundingForms()'>Update Funding</a>
```

Comment:
a couple of things here:

1. this should be a `<button>` instead of an `<a>` tag. that way you don't have to add that href and you can just attach the click event to the button.
2. instead of attaching the `submitFundingForms` function to the onclick, what about adding an id to this element and in your javascript you can grab that id to add an event listener for click events? [this shows an example of how to do that](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#examples). it has the added benefit of keeping all of the javascript contained within the scripts.

Reply:
ohh nice, both of these make a lot of sense.